fish-rust/src/wchar_ext.rs

```diff
@@ -274,8 +274,8 @@ impl WExt for wstr {
 
 #[cfg(test)]
 mod tests {
-    use super::WExt;
-    use crate::wchar::{wstr, WString, L};
+    use super::*;
+    use crate::wchar::L;
```

Comment:
I guess this one can go as well

Reply:
`L` is not imported in the `wchar_ext` mod, and I somewhat wanted to avoid circular imports